tl-tl/node/setup.md

```diff
@@ -16,23 +16,23 @@ The deployment consensus node uses the `Neo-CLI`, a cross-platform program that
 | openSUSE 13.2, 42.1               | ✅                 |
 | Docker                            | ✅                 |
 
-> [!Note]
-> At present, NEO nodes do not work properly under Mac OS. In the future there will be support for Mac OS, please wait for further updates.
+> [!Paalala]
+> Sa kasalukuyan, Ang mga node ng NEO ay hindi gumagana ng maayos sa ilalim ng Mac OS. Sa hinaharap magkakaroon ng suporta para sa Mac OS, kung maaari maghintay para sa mga karagdagang update.
 
-## Run the environment
+## I-run ang environment
 
-Running of an NEO node requires installation of [.NET Core Runtime](https://www.microsoft.com/net/download/core#/runtime), version 1.0.1 or above.
+Sa pagpapatakbo ng isang node ng NEO ay kinakailangan na mag-install ng [.NET Core Runtime](https://www.microsoft.com/net/download/core#/runtime), bersyon 1.0.1 o higit pa.
 
-### Windows system installation method
+### Paraan ng pag-install sa sistem ng Windows
 
-In the Windows system, installation of . NET Core is very convenient, you can directly download and run.
+Sa sistem ng Windows, ang pag-install ng . NET Core ay napakamadali, maaari mong direktang i-download at i-run.
```

Comment:
sistem -> sistema
pag-install -> pag-iinstall
napakamadali -> napakadali

Reply:
i-run to patakbuhin